src/Resources/assets/js/contao-tinymce-bundle.js

```diff
@@ -29,16 +33,142 @@ class TinyMceBundle {
             }
 
             utilsBundle.util.runRecursiveFunction(importPlugin, ['paste', 'link', 'lists'], () => {
-                config.selector = '#' + element.id;
-
-                // set language
-                config.language_url = (typeof NON_ENCORE !== 'undefined' && NON_ENCORE ? '/bundles/heimrichhannottinymce/js/languages/' : '/build/tinymce/languages/') + language + '.js';
-                config.language = language;
+                // modify config before initialization
+                config = TinyMceBundle.modifyConfig(element, config);
 
                 // init
                 tinymce.init(config);
+
+            });
+
+            let form = element.closest('form');
+            if(!form.classList.contains('tinymce-form')) {
+                form.classList.add('tinymce-form');
+            }
+        });
+
+    }
+
+    static initEventListeners() {
+        utilsBundle.event.addDynamicEventListener('click', '.tinymce-limitChars', (element) => {
```

Comment:
Please take care of codestyle: no camelcase in css classes, i.e. tinymce-limit-chars. Please check the rest of the code for this kind of issue.